### PR TITLE
v0.14.5 Update nsb-dev [BAC-2328]

### PR DESCRIPTION
**Pull request by esemeniuc**

### CODE REVIEW

**Review by eyqs — APPROVED** [review]
LGTM

File: __tests__/StarkwareLib.test.ts [hunk 1]
```diff
@@ -5,9 +5,6 @@ import { DummyProvider } from './helpers/DummyProvider';
 import { Networks } from '../src/types';
 import Web3 from 'web3';
 
-// async ropsten calls can take >10k MS
```
Comment (eyqs) [inline __tests__/StarkwareLib.test.ts:8]: Since the default in node service base dev is 30s and this is higher, maybe we need to keep it?

Comment (esemeniuc) [reply __tests__/StarkwareLib.test.ts:8]: reverted

Comment (esemeniuc) [reply __tests__/StarkwareLib.test.ts:8]: i thought there would be a large buffer

**Pull request merged**